diff --git a/src/init.ts b/src/init.ts
index 947b902..3c3778f 100644
--- a/src/init.ts
+++ b/src/init.ts
@@ -18,9 +18,7 @@ function init(
   }
   initCalled = true
   remote.add_interface("testorio-tests-available-for-" + script.mod_name, {})
-  if (script.mod_name === settings.global["testorio:test-mod"].value) {
-    require("@NoResolution:__testorio__/_testorio")(files, config)
-  }
+  require("@NoResolution:__testorio__/_testorio")(files, config)
 }
 
 export = init
